build.gradle

```diff
@@ -40,7 +40,7 @@ targetCompatibility = 1.7
 ext {
   generatedSrcDir = "${projectDir}/generated/src"
 
-  grpcVersion = '1.0.3'
+  grpcVersion = '1.2.0'
```

Comment:
@shinfan Do you want to handle both GCJ and gax? If so, I'll let you upgrade and I'll just rebase.